modules/ROOT/pages/permissions-by-product.adoc

```diff
@@ -34,111 +47,281 @@ Or:
+* Download a copy of the unified schema from the query editor
+
+Operate:: Enables a user to: 
++
+* View customer-facing logs 
+* Set a dedicated load balancer URL for Anypoint Datagraph
+
+DataGraph Admin:: Enables a user to:
++
+* Contribute, consume, and operate the unified schema
+* View usage metrics
+
+DataGraph Project - Contributor:: Enables a user to:
++
+* Add source APIs to the unified schema
+* Edit any source API schema added to the unified schema
+* Request access to run queries
+* Promote API schemas to an environment
+* View query traces in real time while running queries from the UI
+* Download a copy of the unified schema from the query editor in a specific project
+
+DataGraph Project - Operator:: Enables a user to view customer-facing logs and set a dedicated load balancer URL for Anypoint Datagraph in a specific project.
+
+DataGraph Project - Admin::  Enables a user to: 
++
+* Contribute, consume, and operate the unified schema
+* View usage metrics in a specific project
 
 == Design Center
 
-Organization level permissions:
+Design Center Developer:: Enables a user to view, create, and manage all projects within a business group. +
+Use this permission to set up administrators for all the projects within a specific business group.
+Design Center Creator:: Enables a user to create projects in Design Center from the navigation panel and view all projects created or shared with the user. +
+Use this permission to invite users to create, edit, and maintain your projects.
+Design Center Viewer:: Enables a user to view all Design Center projects within a business group and test projects with the Mocking Service. +
+Users with this permission cannot create new projects, edit or rename existing projects, or share projects with another user. 
++
+Assign this permission to those who consume your project in a specific business group.
 
-* Design Center Developer
-* Design Center Creator
-* Design Center Viewer
+Design Center also has project level permissions: 
 
-Project level permissions:
+Project Administrator:: Enables a user to manage and share a Design Center project within a business group. +
+Use this permission to set up administrators for all the projects within a specific business group.
+Project Editor:: Enables a user to edit a Design Center project within a business group. +
+Use this permission to invite users to create, edit, and maintain your projects.
+Project Viewer:: Enables a user to view a Design Center project within a business group and test projects with the Mocking Service. +
+Users with this permission cannot create a new project, edit or rename the existing project, or share the project with another user. 
++
+Assign this permission to those who consume your project in a specific business group.
 
-* Project Administrator
-* Project Editor
-* Project Viewer
+// where did the project-level permissions go?
 
 == Exchange
 
-Organization level permissions:
+Exchange Administrator:: Enables users to: 
```

Comment:
Most of these have "Enables a user to...". Gllobal: this should be consistent.

Reply:
```suggestion
Exchange Administrator:: Enables a user to: 
```